terraform/modules/ops/testing.tf

```diff
@@ -50,10 +50,11 @@ resource "google_cloudbuild_trigger" "api_unit_tests" {
 ###################
 
 resource "google_cloudbuild_trigger" "web_system_tests" {
-  project  = var.project_id
-  count    = var.setup_cd_system ? 1 : 0
-  name     = "web-system-tests"
-  filename = "ops/web-e2e.cloudbuild.yaml"
+  project     = var.project_id
+  count       = var.setup_cd_system ? 1 : 0
+  name        = "testing-web-e2e-test"
```

Comment:
This name change goes from <short component>-<testing objective> to <activity>-<short component>-<objective>. 

Since this will be harder to change later, could we nit pick just a bit and talk about why this is better?

Going back to #669 and contrasting with our components, I don't see "testing" as a scope or component. Is there a goal of visually separating or sorting pure "testing" activities from delivery triggers?

Reply:
- The original (`web-system-tests`) is too broad if there's going to be more than one kind of test for our web component.  
- You're right, testing only makes sense if there was a clear reason to sort by `testing` - which I don't think there is. Based on `[scope]-[outcome]` or `<short component>-<build objective>` would `web-e2e-test` make more sense?

Reply:
Small nit pick, may we leave this as `web-e2e-test`. Would like to have the naming to either all start with verbs or nouns. The back and forth for our resources is a little confusing.